src/components/Dialog-Window/Resources/DialogResources.module.scss

```
@@ -0,0 +1,23 @@
+.li,
+a {
+  color: rgba(31, 103, 199, 1);
+}
+
+.input {
+  width: 228px;
+  height: 32px;
```

Use rems instead of px values for text height values. Some users might have set all font sizes to 400%, which would not fit within 32px of height.

```suggestion
  height: 2rem;
  width: 14rem;
```